### PR TITLE
Support for deletion of multiple files

### PR DESCRIPTION
**Pull request by OrangeJuiced**
This pull request contains the code that supports deletion of multiple files.

### CODE REVIEW

File: src/controllers/fs.js [hunk 1]
```diff
@@ -195,11 +195,19 @@ class FileSystem {
     }
 
     delete(path, next) {
-        // Safety - prevent deleting the main folder.
-        if (Path.resolve(this.server.path(path)) === this.server.path()) {
-            return next(new Error('You cannot delete your home folder.'));
+        var res = path.split(",")
```
Comment (DaneEveritt) [inline src/controllers/fs.js]: This should use lodash's `_.split` function over native.

File: src/controllers/fs.js [hunk 2]
```diff
@@ -195,11 +195,19 @@ class FileSystem {
...
+        var i;
```
Comment (DaneEveritt) [inline src/controllers/fs.js]: `var` is not used in this project, please use `let` here, and `const` where needed elsewhere.

File: src/controllers/fs.js [hunk 3]
```diff
@@ -195,11 +195,19 @@ class FileSystem {
...
+
+        for (i = 0; i < res.length; i++) {
```
Comment (DaneEveritt) [inline src/controllers/fs.js]: This should use `Async.each` operations ideally.

**Review by DaneEveritt — CHANGES_REQUESTED** [review]
Overall looks pretty good, I think this might be better suited to work more like `copy` or other functions that accept an array of files to work with.

If we do that, we need to change the API endpoint, which is something I am okay with so long as the panel is also updated.

**Comment by OrangeJuiced** [comment]
I have changed the endpoint and rewritten the delete function to more closely resemble `copy`.

File: src/http/routes.js [hunk 4]
```diff
@@ -201,11 +206,6 @@ RestServer.post('/server/file/save', (req, res, next) => {
     return next();
 });
 
-RestServer.del(/^\/server\/file\/f\/(.+)/, (req, res, next) => {
```
Comment (DaneEveritt) [inline src/http/routes.js:204]: This is a breaking change. Please revert it or confirm that no other locations in the panel hit this endpoint.

Comment (OrangeJuiced) [reply src/http/routes.js:204]: I have double checked. I can't find anything else that uses that endpoint.

**Review by DaneEveritt — APPROVED** [review]

**Pull request merged**